routes/products.js

```diff
@@ -0,0 +1,14 @@
+var express = require('express');
+var router = express.Router();
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.get('/kufar', (req, res) => {
```

Comment:
'/products'?

Reply:
the whole '/products' route is handled above.
Here we can use simply '/'

Reply:
fixed